src/api/init.ts

```diff
@@ -402,7 +382,7 @@ export async function startApiServer(opts: {
       logger.info('Closing WebSocket channels...');
       ws.close(error => {
         if (error) {
-          logError('Failed to gracefully close WebSocket channels', error);
+          logger.error('Failed to gracefully close WebSocket channels', error);
```

Comment:
In Pino, the first argument should be the object you want to log:
```suggestion
          logger.error(error, 'Failed to gracefully close WebSocket channels');
```
Can you make this change in all logs?